Code/GraphMol/ForceFieldHelpers/Wrap/rdForceFields.cpp

```diff
@@ -143,7 +143,7 @@ ForceFields::PyForceField *MMFFGetMoleculeForceField(
   return pyFF;
 }
 
-bool MMFFHasAllMoleculeParams(ROMol &mol) {
+bool MMFFHasAllMoleculeParams(ROMol mol) {
```

Comment:
I like the idea of doing a copy here instead of actually modifying the molecule.
The way of doing it doesn't match what I think is the usual RDKit approach though: I'd prefer to have this function take a const ROMol reference and to make a copy, so something like:
```
bool MMFFHasAllMoleculeParams(const ROMol &imol) {
   ROMol mol(imol);
   ...
```